sympy/solvers/ode.py

```diff
@@ -373,7 +373,7 @@ def get_numbered_constants(eq, num=1, start=1, prefix='C'):
 
     if isinstance(eq, Expr):
         eq = [eq]
-    elif not is_iterable(eq):
+    elif not iterable(eq):
```

Comment:
Not sure if you wanted feedback yet or not...  but I also think this is the right change